Source/Core/PolygonPipeline.js

```diff
@@ -173,8 +173,7 @@ define([
 
                     i = edges[edge];
                     if (!defined(i)) {
-                        mid = Cartesian3.add(v0, v1, subdivisionMidScratch);
-                        Cartesian3.multiplyByScalar(mid, 0.5, mid);
+                        mid = Cartesian3.midpoint(v0, v1, subdivisionMidScratch);
```

Comment:
The only thing I don't understand is why this `mid` no longer does `Cartesian3.multiplyByScalar(mid, 0.5, mid)`. I thought the `computeSubdivision` function only runs if it's a geodesic, so I'm not sure why this is necessary for this fix.

Reply:
`Cartesian3.midpoint` does both steps internally.
https://github.com/AnalyticalGraphicsInc/cesium/blob/24aa1f7b1a2ef404cc88ebea4020521d7d945f87/Source/Core/Cartesian3.js#L788-L800

The result of `add` is just a intermediate step to getting the midpoint in this case.

Reply:
Oh! I misread that, thought it was still doing `add` without the multiply by half. My bad.